scripts/backup.bety.sh

```diff
@@ -30,49 +29,56 @@ WEEK=$( date +"%W" )
 MONTH=$( date +"%m" )
 YEAR=$( date +"%Y" )
 
-# psql options
-# this allows you to add any other options
+# Psql options
+# This allows you to add any other options
 PG_OPT=${PG_OPT:-""}
 
-# parse command line options
-while getopts hp: opt; do
+# Parse command line options
+while getopts ho:p: opt; do
   case $opt in
   h)
```

Comment:
maybe add an option for `-1` for hourly (every 1 hour).

Reply:
I think adding a '-H' option would make sense. The '-1' option doesn't have a units associated with it and opens the door to '-1h', '-1d', etc.